lib/game.rb

```diff
@@ -0,0 +1,101 @@
+class Game
+  attr_accessor :board, :player_1, :player_2
+  WIN_COMBINATIONS = [
+    [0, 1, 2], 
+    [3, 4, 5], 
+    [6, 7, 8], 
+    [0, 3, 6], 
+    [1, 4, 7], 
+    [2, 5, 8], 
+    [0, 4, 8], 
+    [2, 4, 6]
+    ]
+    
+    def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+      @player_1 = player_1
+      @player_2 = player_2
+      @board = board
+    end
+    
+    def current_player
+      if board.turn_count % 2 == 0
+        player_1
+      else
+        player_2
+      end
+    end
+    
+    def won?
+      WIN_COMBINATIONS.each do |winning_line|
+        if board.cells[winning_line[0]] == "X" && board.cells[winning_line[1]] == "X" && board.cells[winning_line[2]] == "X"
+          return [winning_line[0], winning_line[1], winning_line[2]]
+          elsif board.cells[winning_line[0]] == "O" && board.cells[winning_line[1]] == "O" && board.cells[winning_line[2]] == "O"
+          return [winning_line[0], winning_line[1], winning_line[2]]
+        end
+      end
+      false
+    end
+    
+    def draw?
+      board.full? && !won?
+    end
+    
+    def over?
+      draw? || won?
+    end
+    
+    def winner
+      if won?
+        board.cells[won?[0]]
+      end
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.